prague-postgres-meetup-nov-2024.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Prague PostgreSQL Meetup on November 18th'
+description: 'The Prague PostgreSQL Meetup will take place on November 18th!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/prague-postgres-meetup/prague-postgres-meetup@2x.png
+  alt: Prague PostgreSQL Meetup on November 18th
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 11-18-2024
+tags:
+  [
+    'postgres',
+    'postgresql',
+    'prague-postgres-meetup',
+    'prague',
+    'praha',
+    'meetup',
+    'event',
+    'community',
+    'november',
+    'xata',
+    'p2d2'
+  ]
+published: true
+slug: prague-postgres-meetup-november-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d57njp620sncl5uvdkckvu6a9qr7k
+---
+
+## Prague PostgreSQL Meetup on November 18th
+
+I am happy to announce that the next [Prague PostgreSQL Meetup](https://www.meetup.com/prague-postgresql-meetup/events/303386799) will take place on Monday, November 18th at 17:30 at [Spaces, Nile House](https://www.spacesworks.com/cz/prague-cz/nile-house/) (Karolinská 654/2, 186 00 Praha 8-Rohanský ostrov).
```

Comment:
if this is going live tomorrow, we could say "today" in addition to the actual date